pkg/codegen/nodejs/gen_program.go

```diff
@@ -155,16 +157,24 @@ func GenerateProject(directory string, project workspace.Project, program *pcl.P
 	}
 	files["Pulumi.yaml"] = projectBytes
 
+	if localProjects == nil {
+		localProjects = map[string]string{}
+	}
+
 	// Build the pacakge.json
 	var packageJSON bytes.Buffer
+	pulumi := `@pulumi/pulumi": "^3.0.0`
```

Comment:
```suggestion
	pulumi := `"@pulumi/pulumi": "^3.0.0"`
```
I thought this was a bugged at first glance, simpler to keep all the quotes together rather than wrapping the outer quotes as part of the fmt string below.